src/CMakeLists.txt

```diff
@@ -97,14 +103,25 @@ target_link_libraries( ${LIB_NAME} absl_algorithm )
 target_link_libraries( ${LIB_NAME} absl_container )
 target_link_libraries( ${LIB_NAME} absl_strings )
 
+target_link_libraries( ${LIB_NAME_STATIC} ngraph_logger) 
```

Comment:
should these be in an `if(NGRAPH_BRIDGE_STATIC_LIB_ENABLE)`?

Reply:
yeah will change this, i set NGRAPH_BRIDGE_STATIC_LIB_ENABLE to true right now. gtest linking with static library isn't working yet, trying to get that working first.